addons/account/wizard/account_payment_register.py

```diff
@@ -20,7 +20,7 @@ class AccountPaymentRegister(models.TransientModel):
         compute='_compute_communication')
     group_payment = fields.Boolean(string="Group Payments", store=True, readonly=False,
         compute='_compute_group_payment',
-        help="Only one payment will be created by partner (bank), instead of one per billy.")
+        help="Only one global payment will be created, instead of one per bill.")
```

Comment:
The translation template (`.pot` file) should also be updated in stable versions.
Even though it was asked to do it in 16.0, you also have to count that it will break all the translations until they are translated again. Might not be worth it for such a small change. (also might if it is very misleading)